139_Client_Server/python_server.py

```diff
@@ -0,0 +1,63 @@
+import socket # Gives us #socket.socket() and socket.sock_stream()
+import threading # For multiple connections
+
+clients = []
+
+
+# Thread for each client
+def serve(client_connect, client_address, client_id):
+  global clients
+  try:
+    print(f"Serve {client_id}")
+    while(1):
+      client_connect.sendall(bytes('Write something!',encoding='utf8'))
+      data = client_connect.recv(80)
+      data = data.decode('utf-8')
+      print(f"Got from {client_id} : {data}")
+      if ((data=='quit') or (data=='Quit')):
+        break
+      else:
+        client_connect.sendall(bytes(f'You sent {str(data)}',encoding='utf8'))
+  finally:
+    #Close Connection
+    client_connect.close()
+    clients.remove(client_id)
+    print(clients)
+    print(f"Number of current Connections: {len(clients)}")
+  
+#Function to establish a TCP host
+def server():
+  global clients
+  client_id = 0
+  
+  while(1):
+    print(f"Number of current Connections: {len(clients)}")
```

Comment:
In this case you can also do:
```suggestion
    print("Number of current Connections:", len(clients))
```